src/expandableCalendar/infiniteAgendaList.tsx

```diff
@@ -244,7 +243,7 @@ const InfiniteAgendaList = (props: AgendaListProps) => {
       layoutProvider={layoutProvider}
       onScroll={_onScroll}
       onVisibleIndicesChanged={_onVisibleIndicesChanged}
-      scrollViewProps={{onMomentumScrollEnd: _onMomentumScrollEnd, nestedScrollEnabled: true, refreshControl}}
+      scrollViewProps={{nestedScrollEnabled: true, ...props, onMomentumScrollEnd: _onMomentumScrollEnd}}
```

Comment:
You're passing all the props... this can cause a problem for example to the onScroll we pass the _onScroll and now scrollViewProps gets the props.onScroll. I don't like it. IMO is is better to create a proper const _scrollViewProps that passes only the relevant props with regard to what we want the user to override and what not